resolvers.js

```diff
@@ -255,6 +255,15 @@ const Resolvers = {
 
     uploadDateTime(ds) {
       return ds._source.ds_upload_dt;
+    },
+
+    fdrCounts(ds) {
+      // Get 10% FDR string -> counts[1], 5% -> counts[0]
+      let fdrAt10Val = ds._source.annotation_counts[0].counts[1];
```

Comment:
The implementation is too specific here. One of ways to make it more general and more useful in the future is adding the needed FDR level as a parameter

Reply:
Given the name of the method I'd expect it to return a map of the form fdr_level -> count_n
I thought we want to display this information in the web app on the datasets page

Reply:
I agree with the expectation here, but since we don't really need such a map anywhere, I'd transform it to a bit more specific function

Reply:
Yes, I think the initial idea was to get all FDR levels with number of counts for each of them. However, after discussion with Theo , only 10% FDR counts were chosen to be shown. Adding FDR level as a param would make this code much more flexible. Another option is to return all FDR levels with counts and then decide on the web-app side, which level to show. I think the latter is the better option, becasue then we can allow user to chose for instance for the level himself w/o extra requests to GQL. What do you think @iprotsyuk @intsco ?

Reply:
I guess if there're no other FDR-related feature requests for the web-app which could reuse these GQL endpoints, I wouldn't go for the general solution, as it brings more unnecesary data transfer over the network. We can always add the general method once it's needed. So I'd stick to a parametrical FDR count request for the time being.

Reply:
Got it, thank you @intsco @iprotsyuk 
So, I will send all 4 pairs of FDR with counts to web-app with only 10% displayed at the moment.

Reply:
@Ren22 you could get the best of all worlds and accept an array of fdr values to return as an input...

Reply:
thanks @andy-d-palmer! will do that

Reply:
@Ren22, bear in mind that this approach is a bit complicated with argument handling. What if requested FDR level is not one of the pre-computed ones by SM-engine, including cases when it's outside of [0, 100]% interval. Also there's unnecessary degree of freedom in a sense that values in input array can be repeated. Overall, it seems that returning all available FDR counts is more preferable option here.

Reply:
@iprotsyuk great points about the input handling. I added a feature request for making the backend less coupled to the web app (i.e. storing float fdr values rather than discrete levels for display) so it makes sense to have more general api.